tests/modules/programs/swaylock/settings.nix

```diff
@@ -1,11 +1,13 @@
-{ ... }: {
-  programs.swaylock.settings = {
-    color = "808080";
-    font-size = 24;
-    indicator-idle-visible = false; # Test that this does nothing
-    indicator-radius = 100;
-    line-color = "ffffff";
-    show-failed-attempts = true;
+{ config, ... }: {
+  programs.swaylock = {
+    settings = {
```

Comment:
`settings` does not have to be nested here because the `enable` and `package` options are not being set alongside it any more. However, if it's all right to change this file in this pull request, we could remove `{ ... }:` to simplify it because it does not accept any parameters anyway.

Reply:
Fixed it. :)